app/models/vm.rb

```diff
@@ -5,7 +5,7 @@ class Vm < ApplicationRecord
 
   belongs_to :tenant
   belongs_to :source
-  belongs_to :orchestration_stack
+  belongs_to :orchestration_stack, :optional => true
```

Comment:
ah, could we maybe disable these? We have not null in the DB, so we know what is mandatory